rhobot.go

```diff
@@ -14,66 +15,111 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = "Rhobot"
-	app.Usage = "Rhobot is your friend."
+	app.Usage = "Rhobot is a database development tool that uses DevOps best practices."
 	app.EnableBashCompletion = true
 
+	config := config.NewConfig()
+
 	app.Commands = []cli.Command{
 		{
 			Name:    "run",
 			Aliases: []string{},
-			Usage:   "healthchecks|gocd|tbd",
+			Usage:   "healthchecks|pipeline|tbd",
 			Subcommands: []cli.Command{
 				{
 					Name:  "healthchecks",
-					Usage: "[database uri] [path to healthcheck file]",
+					Usage: "HEALTHCHECK_FILE [DATABASE_URI]",
```

Comment:
Good and proper usage statement indicating proper optional arguments. I like it
http://courses.cms.caltech.edu/cs11/material/general/usage.html